packages/scoutgame-ui/src/components/common/Card/BuilderCard/BuilderCardActivity/BuilderCardActivityTooltip.tsx

```diff
@@ -1,64 +1,46 @@
 import { Stack, Typography } from '@mui/material';
 
-import { GemsIcon } from '../../../Icons';
-
 export function BuilderCardActivityTooltip() {
   return (
-    <Stack flexDirection='column' gap={0.5} py={1} width='fit-content'>
-      <Stack flexDirection='row' gap={2} justifyContent='space-between' alignItems='center'>
-        <Stack width={50} alignItems='center'>
-          <Stack
-            sx={{
-              width: 20,
-              height: 20,
-              borderRadius: '50%',
-              backgroundColor: 'text.secondary'
-            }}
-          />
-        </Stack>
-        <Stack flexDirection='row' gap={0.5} flex={1} alignItems='center'>
-          <Typography>= Scored 30+ </Typography>
-          <GemsIcon size={16} />
-        </Stack>
+    <Stack flexDirection='column' gap={2.5} width='fit-content'>
+      <Stack>
+        <Typography fontWeight='bold'>WEEK’S GEMS</Typography>
+        <Typography>
+          Number of Gems earned by contributing to approved projects for the current week. Developers are ranked by
+          number of Gems earned at the end of each week. Higher rank leads to more Scout Points.
+        </Typography>
+      </Stack>
+
+      <Stack>
+        <Typography fontWeight='bold'>LEVEL</Typography>
+        <Typography>
+          Represents the percentile of the Developer's weekly Scout Point average relative to the point average of all
+          active developers. The top 10% of Developers are Level 10.
```

Comment:
Suggestion for copy;

Represents the **decile** of the Developer's weekly github contributions amongst active developers. The top 10% of Developers are Level 10.

Reply:
updated